oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/delegate/PropertyDelegate.java

```diff
@@ -92,7 +92,7 @@ public Status getStatus() {
 
     @Override
     public boolean isProtected() throws InvalidItemStateException {
-        return getParent().isProtected(name);
+        return getParent().isProtected(name, state.getType());
```

Comment:
`state` may be `null` when the property is stale. E.g. another session removed the property.

```suggestion
        return getParent().isProtected(name, getPropertyState().getType());
```
The implementation actually has another similar problem. `getParent()` may also return null :-/ I'll create a separate issue for that.